org-formation/075-security-hub/security-hub-suppress-infra.yaml

```diff
@@ -382,3 +382,39 @@ Resources:
           - SecurityHubFindingsQueue
           - Arn
         Id: Target0
+
+  # This rule suppresses findings with the given generator IDs only for the bootstrap account and service accounts
+  SuppressFindingsForPublicBucketsRule:
+    Type: AWS::Events::Rule
+    Properties:
+      Description: SecHubSuppress findings (identified by generatorId) for the bootstrap and essentials buckets in all accounts
+      EventPattern:
+        detail:
+          findings:
+            Resources:
+              Id:
+                # synapsedw IT-2308, do not exempt human users
+                - prefix: 'arn:aws:iam::383874245509:user/bootstrap-AWSIAMTravisUser-'
```

Comment:
Is this account deprecated? Looking it either does not exist or has not been used in a long while (except scicomp/scipool).

Reply:
i believe that is the `org-sagebase-synapsedw` account.  in any case `bootstrap-AWSIAMTravisUser-*` is old and should not be used anymore.  if it's still around then it should probably just be deleted